api/v1alpha2/linodevpc_types.go

```diff
@@ -54,6 +53,9 @@ type VPCSubnetCreateOptions struct {
 	Label string `json:"label,omitempty"`
 	// +optional
 	IPv4 string `json:"ipv4,omitempty"`
+	// ID is subnet id for the subnet
+	// +optional
+	ID int `json:"id,omitempty"`
```

Comment:
question: would it make more sense to call this subnetID?

Reply:
sorry to chime in on this late, I don't have a strong opinion here(and I think we break this other places as well) but for what it's worth `subnet.subnetID` is redundant but I'm fine with keeping it if that is the standard we have elsewhere, just wanted to call that out